src/java.base/share/classes/java/lang/ref/NativeReferenceQueue.java

```diff
@@ -52,41 +58,4 @@ void await() throws InterruptedException {
     void await(long timeoutMillis) throws InterruptedException {
```

Comment:
I don't expect these methods are required either, they support the OpenJDK implementation (although they shouldn't cause any problems today). Only the constructor is needed.

Reply:
done